pages/about/index.md

```diff
@@ -1,62 +1,76 @@
+
+One of the great things about a large government organization is that we're not motivated by profit. "The greater good" might sound like a cliché, but it's true. LLNL's national security mission governs our work, so our OSS portfolio of apps, libraries, compilers, and other tools support that mission. This includes everything from monitoring the performance of our supercomputers to making multiphysics codes run more smoothly.
+
+Case in point: The DOE's [Advanced Simulation and Computing (ASC) Program](https://asc.llnl.gov) leverages OSS to build, field, and integrate Linux clusters into production service at our [Livermore Computing high-performance computing center](https://hpc.llnl.gov/). ASC is responsible for developing computer simulation capabilities that analyze and predict the performance, safety, and effectiveness of the nation's nuclear stockpile. In other words, the stakes are high for ensuring our code is accurate. With OSS, we can develop quickly and choose the best solutions.
+
+This portal abounds with examples of projects that have found a home in the open source community. Access is key, which is why we jumped at the chance to index our software on [Code.gov](https://code.gov){% raw %}{% endraw %}. You'll find our work in the DOE [repo list](https://code.gov/#!/browse-projects?agencies=DOE){% raw %}{% endraw %}, and we invite you to browse our projects on [GitHub](https://github.com/LLNL){% raw %}{% endraw %}.
+
+\* The original policy memorandum is no longer available online. For details, please see 
+<!-- START: Buttons -->
+* [The People's Code Blog Post](https://www.cio.gov/2016/08/11/peoples-code.html){% raw %}{% endraw %}
+* [The Archived Policy on GitHub](https://github.com/WhiteHouse/source-code-policy){% raw %}{% endraw %}
+<!-- END: Buttons -->
+
+## Learn More About OSS at LLNL
+<!-- START: Quicklinks boxes -->
+* [LLNL Software News Archive](/news/archive/)
+* [The Case for Open Source Software](https://18f.gsa.gov/2018/07/12/the-case-for-open-source-software/){% raw %}{% endraw %}
+* [OSS Spotlight: LLNL and ZFS on Linux](https://medium.com/codedotgov/oss-spotlight-lawrence-livermore-national-laboratory-and-zfs-on-linux-6596fca6e5f6){% raw %}{% endraw %}
+* [Spack, A Lab-Developed ‘App Store for Supercomputers,’ becoming Standard-Bearer](https://www.llnl.gov/news/spack-lab-developed-app-store-supercomputers-becoming-standard-bearer){% raw %}{% endraw %}
+<!-- END: Quicklinks boxes -->
+
+### *Science & Technology Review* coverage
+<!-- START: Quicklinks boxes -->
+* [Ambassadors of Code (cover story)](https://str.llnl.gov/2018-01/lee)
+* [Ambassadors of Code (video, 3:04)](https://youtu.be/nTxMn1NWHQU){% raw %}{% endraw %}
+* [Commentary by Bruce Hendrickson](https://str.llnl.gov/2018-01/comjan18), LLNL Computing associate director
+* [Visualization Software Stands the Test of Time](https://str.llnl.gov/2021-05/brugger)
+<!-- END: Quicklinks boxes -->
+  
+### 2020 LLNL Computing Virtual Expo Materials
```

Comment:
@cyndy-llnl What do you think about removing "2020" from this heading so it doesn't seem so outdated? The assets from the Expo are fairly evergreen.

Reply:
@hauten That works! I'm a fan of including the date if something is "of a moment," but when it's evergreen, you'll entice more folks to use it when an older date is omitted. 

One thing: Is that a change you're making or that you'd like me to make? Happy to participate however you prefer!

Reply:
@cyndy-llnl I made the change just now, thx